lib/cli/src/generators/WEB-COMPONENTS/template/stories/0-Welcome.stories.js

```diff
@@ -0,0 +1,49 @@
+import { html } from 'lit-html';
+
+export default {
+  title: 'Welcome',
+};
+
+export const ToStorybook = () => html`<main>
+  <h1>Welcome to storybook</h1>
+  <p>This is a UI component dev environment for your app.</p>
+  <p>
+    We've added some basic stories inside the
+    <span class="inline-code">src/stories</span> directory. <br />
+    A story is a single state of one or more UI components. You can have as many stories as you
+    want. <br />
+    (Basically a story is like a visual test case.)
+  </p>
+  <p>
+    See these sample
+    <a (click)="showApp.emit($event)" role="button" tabindex="0">stories</a> for a component called
+    <span class="inline-code">Button</span> .
+  </p>
+  <p>
+    Just like that, you can add your own components as stories. <br />
+    You can also edit those components and see changes right away. <br />
+    (Try editing the <span class="inline-code">Button</span> stories located at
+    <span class="inline-code">src/stories/index.js</span>.)
+  </p>
+  <p>
+    Usually we create stories with smaller UI components in the app.<br />
+    Have a look at the
+    <a
+      href="https://storybook.js.org/basics/writing-stories"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Writing Stories
+    </a>
+    section in our documentation.
+  </p>
+  <p class="note">
+    <b>NOTE:</b> <br />
+    Have a look at the <span class="inline-code">.storybook/webpack.config.js</span> to add webpack
+    loaders and plugins you are using in this project.
+  </p>
```

Comment:
I think we can remove that as I'm not sure there will be a `.storybook/webpack.config.js`

Reply:
You are right, the template does not contain such file:
![image](https://user-images.githubusercontent.com/1671563/81496423-e365ee00-92b7-11ea-98eb-239c300ebd7f.png)

However it's nice to tell people that if they add a `webpack.config.js` file it will be picked up, so maybe just rephrase that? Also, that sentence is in every framework's template, so if we were to make a change, it has to be synced with the others.

@shilman what's your take on this?

Reply:
AFAIK the current recommendation is to override `webpackFinal` in `main.js`, so we should probably update all the docs accordingly. `webpack.config.js` is still supported (for now, not sure how long we plan to keep this? But we should be  directing everybody towards the recommendation. @ndelangen please feel free to correct if I got any of this wrong.

Reply:
> the current recommendation is to override webpackFinal in main.js

indeed